conf/base.config

```diff
@@ -7,6 +7,16 @@
     the PATH. Runs in `local` mode - all jobs will be run on the logged in environment.
 ----------------------------------------------------------------------------------------
 */
+// Set default parameters that may be overwritten
+params {
+    download_annots = 'none'
```

Comment:
These should be in the `nextflow.config` file not here. That's where all the inputs are defined and we should keep that consistent.

Reply:
Ahhh, okay - I wasn't sure where best to put these, but it worked so I stuck with it. Can move them easily though.

Reply:
Don't forget to move these.

Reply:
Done.